decimal64.go

```diff
@@ -495,3 +517,34 @@ func propagateNan(dp, ep *decParts) *Decimal64 {
 	}
 	return ep.dec
 }
+
+// Class takes one operand and provides the class the decimal is in
```

Comment:
Public symbols comments should be full sentences (i.e., with a period) for `go doc` readability.